core/api/src/graphql/public/root/mutation/onchain-payment-send.ts

```diff
@@ -32,7 +32,7 @@ const OnChainPaymentSendMutation = GT.Field<
     input: {
       walletId: WalletId | InputValidationError
       address: OnChainAddress | InputValidationError
-      amount: number
+      amount: number | InputValidationError
```

Comment:
why are not we using SatAmount? https://github.com/GaloyMoney/galoy/blob/main/core/api/src/graphql/shared/types/scalar/sat-amount.ts

Reply:
I think we are?
https://github.com/GaloyMoney/galoy/blob/7352f934d6739983d224665fcc57977e6536af6d/core/api/src/graphql/public/root/mutation/onchain-payment-send.ts#L19